src/Makefile.in

```diff
@@ -337,7 +337,7 @@ host_cpu = @host_cpu@
 host_os = @host_os@
 host_vendor = @host_vendor@
 htmldir = @htmldir@
-includedir = /usr/local/include
+includedir = @includedir@
```

Comment:
The only issue I have with this solution is that this file was generated during the automake process, meaning it will be overwritten when running automake again. I've found a way to fix these issues, and can commit them to this brach if you wish.